src/main/java/org/moe/maven/GradleTask.java

```diff
@@ -140,8 +140,11 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             final ByteArrayOutputStream baos = new ByteArrayOutputStream();
             if (!isPrintTask()) {
                 launcher.setStandardOutput(baos);
+                launcher.setStandardError(baos);
             } else {
-                launcher.setStandardOutput(new ConsolePrinter());
+                final ConsolePrinter console = new ConsolePrinter();
+                launcher.setStandardOutput(console);
+                launcher.setStandardError(console);
```

Comment:
With stderr it is finally possible to see what is the reason for `moe:launch` task failing.